server/main.lua

```diff
@@ -0,0 +1,101 @@
+---@class vehicleData
```

Comment:
```suggestion
---@class VehicleData
```
capitalize class names